### DISCUSSION TITLE
imgur: uploaded images don't show up in account

### DISCUSSION THREAD

**Discussion by qoreQyaS** [flameshot-org/flameshot#3004] [Q&A] [answered]
unsure if bug so posting here.
Using version 12.1.0-1 on debian.
I created an appplication ID on the API website of imgur, set it up in the client and uploaded an image to imgur. everything works fine but shouldn't the image show up in my uploaded images in my imgur account?
same after restarting the client, the setting is fine, my client ID shows up there.

**Comment by mmahmoudian** [comment, marked as answer]
No it should not show in your account. This is how imgur works. Perhaps reading the imgur documentation can clarify the situation further for you.
